pkg/task/task.go

```diff
@@ -35,7 +35,7 @@ func DoRetryWithTimeout(t func() (interface{}, error), timeout, timeBeforeRetry
 					return
 				}
 
-				util.Infof("%v. Retry count: %v Next retry in: %v", err, count, timeBeforeRetry)
+				util.Warnf("%v. Retry count: %v Next retry in: %v", err, count, timeBeforeRetry)
```

Comment:
Keep it info. Since this routine is used to operations like reboot, driver bounce where it is expected to do retries. Adding a warning will create a false alarm when someone is debugging a different issue.